test/spec/adapters/smartadserver_spec.js

```diff
@@ -0,0 +1,117 @@
+describe("smartadserver adapter tests", function () {
+  var urlParse = require("url-parse");
+  var querystringify = require("querystringify");
+  var adapter = require("src/adapters/smartadserver");
+  var adLoader = require("src/adloader");
+  var expect = require("chai").expect;
+  var bidmanager = require("src/bidmanager");
+  var CONSTANTS = require('src/constants.json');
+
+  var DEFAULT_PARAMS = {
+      bidderCode: "smartadserver",
+      bids: [{
+        bidId: "abcd1234",
+          sizes: [[300, 250], [300, 200]],
+            bidder: "smartadserver",
+            params: {
+              domain: "http://www.smartadserver.com",
+              siteId: "1234",
+              pageId: "5678",
+              formatId: "90",
+              target: "test=prebid"
+            },
+            requestId: "efgh5678",
+            placementCode: "sas_42"
+        }
+      ]
+  };
+
+  var BID_RESPONSE = {
+    cpm: 0.42,
+    ad: "fake ad content",
+    width: 300,
+    height: 250
+  };
+
+  it("set url parameters", function () {
+    var stubLoadScript = sinon.stub(adLoader, "loadScript");
+
+    adapter().callBids(DEFAULT_PARAMS);
+
+    var smartCallback;
+    for (var k in $$PREBID_GLOBAL$$) {
+      if (k.startsWith("sas_")) {
```

Comment:
Here's the line causing the failure. [This page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) has a polyfill that could be used directly or adapted to make this work cross-browser